atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1804,6 +1804,19 @@ void WebContents::SendInputEvent(v8::Isolate* isolate,
             mouse_wheel_event);
 #endif
       } else {
+        // Populate phase info for the wheel event, otherwise DCHECK
+        // will fail in Chromium. See: CR issue 756524
```

Comment:
Would be better to add a link here https://crbug.com/756524. Not a big deal though.

Reply:
Added.